ms2rescore/retention_time.py

```diff
@@ -23,7 +23,7 @@ def __init__(
         peprec_path: str,
         feature_path: str,
         higher_psm_score_better: bool = True,
-        calibration_set_size: Optional[Union[int, float]] = 0.20,
+        calibration_set_size: Optional[Union[int, float]] = 0.10,
```

Comment:
We should make this a configurable option (through deeplc section in config file).

I can take care of this.